src/test/scala/nl/knaw/dans/easy/stage/EasyStageDatasetSpec.scala

```diff
@@ -69,34 +83,36 @@ class EasyStageDatasetSpec extends FlatSpec with Matchers {
     implicit val s = createSettings(bagitDir, sdoSetDir)
 
     createFileAndFolderSdos(dataDir, DATASET_SDO, OPEN_ACCESS_FOR_REGISTERED_USERS) shouldBe a[Success[_]]
-    FileUtils.readFileToString(fileMetadataFile) should include ("<visibleTo>KNOWN</visibleTo>")
-    FileUtils.readFileToString(fileMetadataFile) should include ("<accessibleTo>KNOWN</accessibleTo>")
+    readFileToString(fileMetadataFile) should include ("<visibleTo>KNOWN</visibleTo>")
```

Comment:
Any specific reason why you don't like the `FileUtils` class to be explicitly here and do use it in the test above?